src/main/java/io/confluent/idesidecar/restapi/clients/SchemaErrors.java

```diff
@@ -25,23 +46,44 @@ public record Error(String message) {
 
   }
 
-  private static final Duration SCHEMA_FETCH_ERROR_TTL = Duration.ofSeconds(30);
+  @ConfigProperty(name = "ide-sidecar.schema-fetch-error-ttl")
+  long schemaFetchErrorTtl;
```

Comment:
This still can be static and final, and we can still have this be a `Duration` type. See other examples in the codebase.